### DISCUSSION TITLE
Is Obtainium also made for Google TV

### DISCUSSION THREAD

**Discussion by Neobond** [ImranR98/Obtainium#1806] [Q&A]
Might be a dumb question but the screenshots suggest it is made for Android phones. Just wondering if the UI also supports Smart TVs (Android TV / Google TV) Thanks.

**Comment by ImranR98** [comment]
You could install it, but there are issues with text input - see #281.

**Comment by Neobond** [comment]
Thanks, I will install it. I have a bluetooth mouse attached to the TV anyway :)
